framework/src/modules/network/network.js

```diff
@@ -37,20 +40,32 @@ module.exports = class Network {
 
 		this.logger = createLoggerComponent(loggerConfig);
 
-		const initialNodeInfo = await this.channel.invoke('chain:getNodeInfo');
+		// TODO: Nonce overwrite should be removed once the Network module has been fully integreated into core and the old peer system has been fully removed.
+		// We need this because the old peer system which runs in parallel will conflict with the new one if they share the same nonce.
+		const moduleNonce = randomstring.generate(16);
+		const sanitizeNodeInfo = nodeInfo => (
+			{
+				...nodeInfo,
+				state: 2, // TODO: Delete state property
+				nonce: moduleNonce,
```

Comment:
Shouldn't we use nonce coming from the config? `this.options.config.nonce`?

Reply:
Eventually yes, but for now in order for it to work with both peer systems, we need each one to have a different nonce that way peers will treat us as if we are two different nodes.